src/components/scenes/Fio/FioAddressSettingsScene.tsx

```diff
@@ -142,42 +145,48 @@ export class FioAddressSettingsComponent extends React.Component<Props, LocalSta
     const { showTransfer, showAddBundledTxs } = this.state
+    const styles = getStyles(theme)
 
     return (
-      <SceneWrapper>
-        <RowUi4 title={lstrings.fio_address_register_form_field_label} body={fioAddressName} />
-        {bundledTxs != null ? <RowUi4 title={lstrings.fio_address_details_screen_bundled_txs} body={`${bundledTxs}`} /> : null}
-        {showAddBundledTxs && (
-          <FioActionSubmit
-            onSubmit={this.onAddBundledTxsSubmit}
-            onSuccess={this.afterAddBundledTxsSuccess}
-            getOperationFee={getAddBundledTxsFee}
-            successMessage={lstrings.fio_request_add_bundled_txs_ok_text}
-            onCancel={this.cancelOperation}
-            fioWallet={fioWallet}
-            addressTitles
-            showPaymentWalletPicker
-            navigation={this.props.navigation}
-          />
-        )}
-        {showTransfer && (
-          <FioActionSubmit
-            goTo={this.goToTransfer}
-            getOperationFee={this.getTransferFee}
-            fioWallet={fioWallet}
-            addressTitles
-            navigation={this.props.navigation}
-          />
-        )}
-        {!showAddBundledTxs && !showTransfer && (
-          <>
-            <MainButton label={lstrings.title_fio_add_bundled_txs} onPress={this.onAddBundledTxsPress} marginRem={[1.5, 1, 0.25]} />
-            <MainButton label={lstrings.title_fio_transfer_address} onPress={this.onTransferPress} marginRem={[0.25, 1]} />
-          </>
-        )}
+      <SceneWrapper scroll>
+        <SceneHeader title={lstrings.title_fio_address_settings} underline withTopMargin />
+        <View style={styles.container}>
+          <CardUi4 sections>
+            <RowUi4 title={lstrings.fio_address_register_form_field_label} body={fioAddressName} />
+            {bundledTxs != null ? <RowUi4 title={lstrings.fio_address_details_screen_bundled_txs} body={`${bundledTxs}`} /> : null}
+          </CardUi4>
+          {showAddBundledTxs && (
+            <FioActionSubmit
+              onSubmit={this.onAddBundledTxsSubmit}
+              onSuccess={this.afterAddBundledTxsSuccess}
+              getOperationFee={getAddBundledTxsFee}
+              successMessage={lstrings.fio_request_add_bundled_txs_ok_text}
+              onCancel={this.cancelOperation}
+              fioWallet={fioWallet}
+              addressTitles
+              showPaymentWalletPicker
+              navigation={this.props.navigation}
+            />
+          )}
+          {showTransfer && (
+            <FioActionSubmit
+              goTo={this.goToTransfer}
+              getOperationFee={this.getTransferFee}
+              fioWallet={fioWallet}
+              addressTitles
+              navigation={this.props.navigation}
+            />
+          )}
+          {!showAddBundledTxs && !showTransfer && (
+            <>
+              <MainButton label={lstrings.title_fio_add_bundled_txs} onPress={this.onAddBundledTxsPress} marginRem={[0.25, 0, 0.25]} />
```

Comment:
2 primary buttons. Use ButtonsView

Reply:
There's no way to do 2 primary buttons in `ButtonsView`

Reply:
Yes, that was by design. I meant that 2 primary buttons according to most of the literature I've read is bad UX. Similar to how we did away with dual secondary buttons.

I think it's safe to assume that adding bundled tx's is the primary action here and transfer address is the secondary (or better, tertiary)